vpr/src/route/route_path_manager.h

```diff
@@ -0,0 +1,97 @@
+#include "rr_graph_fwd.h"
+#include "vtr_assert.h"
+
+#include <set>
+#include <list>
+#include <vector>
+
+#ifndef _PATH_MANAGER_H
+#    define _PATH_MANAGER_H
+
+/* Extra path data needed by RCV, seperated from t_heap struct for performance reasons
+ * Can be accessed by a pointer, won't be initialized unless needed
+ *
+ * path_rr: The entire partial path up until the route tree
```

Comment:
Explain what this is.  Entire partial path until we reach either the net SOURCE or a part of the routing tree that has already been created for other connections for this net.